lib/bolt/transport/orch.rb

```diff
@@ -37,7 +37,8 @@ def self.validate(options); end
 
       def initialize(*args)
         @connections = {}
-        super
+        @from_api = args.select { |arg| arg[:from_api] == true }.any?
```

Comment:
We should probably avoid this pattern and subclass the transport. This adds complexity to the live bolt code for a prototype which is bad and long term it seems likely the normal transport and the orchestrator executor will diverge.